### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by fox3000foxy**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -39,22 +40,33 @@ server.listen(port, function () {
...
-        version: config.version,
-    });
+    var opts = {
+        host: settings[1] || config.ip,
+        port: settings[2] || config.port,
+        username: settings[0].split("%C2%A7")[0],
+        version: '1.16.1',
+    }
+    console.log(opts)
+    var bot = mineflayer.createBot(opts);
```
Comment (fox3000foxy) [inline src/index.js:53]: this is for connect like /#drmike§ip§port

Comment (fox3000foxy) [inline src/index.js:53]: There are lines to change

Comment (fox3000foxy) [inline src/index.js:53]: Update : there are the line to change

**Review by fox3000foxy — COMMENTED** [review]
Plz pull

**Comment by michaljaz** [comment]
it has been added already

**Pull request closed without merging**